modules/net-vpc/subnets.tf

```diff
@@ -130,10 +130,10 @@ resource "google_compute_subnetwork" "proxy_only" {
   ip_cidr_range = each.value.ip_cidr_range
   description = (
     each.value.description == null
-    ? "Terraform-managed proxy-only subnet for Regional HTTPS or Internal HTTPS LB."
+    ? "Terraform-managed proxy-only subnet for Regional HTTPS, Internal HTTPS or Cross-Regional HTTPS Internal LB."
     : each.value.description
   )
-  purpose = "REGIONAL_MANAGED_PROXY"
+  purpose = each.value.global != false ? "GLOBAL_MANAGED_PROXY" : "REGIONAL_MANAGED_PROXY"
```

Comment:
This can be rewritten as:
`each.value.global ? "GLOBAL_MANAGED_PROXY" : "REGIONAL_MANAGED_PROXY"`

which is shorter

Reply:
thanks for that - i find that the way i've put this is a little easier for read for peeps without deep TF experience!